src/build_scripts/buildlib.csm_share

```diff
@@ -13,22 +13,22 @@ my ($sharedlibroot, $installroot, $CASEROOT) = @ARGV;
 
 chdir "${CASEROOT}" or die "Could not cd to \"$CASEROOT\"";
 
-my $CIMEROOT		= `./xmlquery CIMEROOT		-value`;
-my $COMP_INTERFACE	= `./xmlquery COMP_INTERFACE	-value`;
-my $USE_ESMF_LIB	= `./xmlquery USE_ESMF_LIB	-value`;
-my $GMAKE_J		= `./xmlquery GMAKE_J		-value`;
-my $GMAKE		= `./xmlquery GMAKE		-value`;
-my $CASETOOLS		= `./xmlquery CASETOOLS		-value`;
-my $NINST_ATM		= `./xmlquery NINST_ATM		-value`;
-my $NINST_ICE		= `./xmlquery NINST_ICE		-value`;
-my $NINST_GLC		= `./xmlquery NINST_GLC		-value`;
-my $NINST_LND		= `./xmlquery NINST_LND		-value`;
-my $NINST_OCN		= `./xmlquery NINST_OCN		-value`;
-my $NINST_ROF		= `./xmlquery NINST_ROF		-value`;
-my $NINST_WAV		= `./xmlquery NINST_WAV		-value`;
-my $NINST_ESP		= `./xmlquery NINST_ESP		-value`;
-my $NINST_VALUE		= `./xmlquery NINST_VALUE	-value`;
-$ENV{PIO_VERSION}  	= `./xmlquery PIO_VERSION	-value`;
+my $CIMEROOT		= `./xmlquery CIMEROOT		--value`;
+my $COMP_INTERFACE	= `./xmlquery COMP_INTERFACE	--value`;
+my $USE_ESMF_LIB	= `./xmlquery USE_ESMF_LIB	--value`;
+my $GMAKE_J		= `./xmlquery GMAKE_J		--value`;
+my $GMAKE		= `./xmlquery GMAKE		--value`;
+my $CASETOOLS		= `./xmlquery CASETOOLS		--value`;
+my $NINST_ATM		= `./xmlquery NINST_ATM		--value`;
+my $NINST_ICE		= `./xmlquery NINST_ICE		--value`;
+my $NINST_GLC		= `./xmlquery NINST_GLC		--value`;
+my $NINST_LND		= `./xmlquery NINST_LND		--value`;
+my $NINST_OCN		= `./xmlquery NINST_OCN		--value`;
+my $NINST_ROF		= `./xmlquery NINST_ROF		--value`;
+my $NINST_WAV		= `./xmlquery NINST_WAV		--value`;
+my $NINST_ESP		= `./xmlquery NINST_ESP		--value`;
+my $NINST_VALUE		= `./xmlquery NINST_VALUE	--value`;
+$ENV{PIO_VERSION}  	= `./xmlquery PIO_VERSION	--value`;
 #--------------------------------------------------------------------
```

Comment:
These changes do not appear to be related to the PR subject. Please either use a separate PR or document the change in the PR text.